macros/src/test/scala/SRAMCompiler.scala

```diff
@@ -0,0 +1,15 @@
+package barstools.macros
+
+import mdf.macrolib._
+
+class SRAMCompiler extends MacroCompilerSpec with HasSRAMGenerator {
```

Comment:
Can we have a test for the new facilities as well to show that it works? I can help with this if needed.

Reply:
Yeah, I guess this doesn't check anything - it would be good to test it. I think it should be straightforward, but I'll take a crack at it later.

Reply:
Great. I just ask this since I caught so many bugs and edge cases by unit testing when I first wrote this code.